src/Dobble/DeckValidator.php

```diff
@@ -52,4 +53,24 @@ private static function validateDeckCardsHaveSameSymbolNumber($deck)
             throw new DobbleException('Cards have different number of symbol');
         }
     }
+
+    private static function validateAllCardHasOnePair($deck)
+    {
+        foreach ($deck->getCards() as $card) {
+            foreach ($deck->getCards() as $pair) {
+                if ($card != $pair) {
+                    $similarSymbols = array_intersect(
+                        $card->getSymbols(),
+                        $pair->getSymbols()
+                    );
+
+                    if (count($similarSymbols) !== 1) {
+                        $error = 'At least one card has not one and only one '
```

Comment:
Write all this error on a single line (even if we go through the limit of 80 chars)